### PR TITLE
Main Menu

### PR DESCRIPTION
**Pull request by kamishettysreeja25**
- [x] consistent indentation in heads
- [x] Create valid HTML code - for ever open tag there should be a closing tag and vice versa.
- [x] favicon used by html files
- [x] titles in html files
- [x]  removed main_menu.js- used direct links wherever necessary 
- [x] "Exit" should to  be "Links"

@niccokunzmann 
This commit has  completed the tasks as you mentioned in the #100 (comment) 
I have added 4 files 
1-main_menu.html - contains the main menu code
2-leaderboard.html-this page is empty - should be developed soon 
3-howtoplay.hmtl- this contains the game rules which should be added
4-main_menu.css - contains the styling for main_menu 	

We can delete the previous pull request #100 
If anything else is missing please tell me :) 
thank you

### CODE REVIEW

File: style/main_menu.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    background-color: #555;
+    color: white;
+}
+#x {
+    position: absolute;
+    width: 50px;
+    margin-top: 10px;
+}
+body {
+    background-color: lightblue;
```
Comment (niccokunzmann) [inline style/main_menu.css]: please add the `margin:0;padding:0;border:0;` here and remove the line I comment on in main_menu.html

File: main_menu.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
+<html>
+    <head>
+        <title>Flappy Main Menu</title>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="icon" type="image/ico" href="images/favicon.ico"/>
+        <link rel="stylesheet" type="text/css" href="style/main_menu.css">
+    </head>
+    <body>
+        <body style="margin:0;padding:0;border:0;">
```
Comment (niccokunzmann) [inline main_menu.html]: this second body tag can be moved to main_menu.css

File: main_menu.html [hunk 3]
```diff
@@ -0,0 +1,54 @@
+<html>
```
Comment (niccokunzmann) [inline main_menu.html]: I am really impressed when looking at this file. It is so tidy :+1:

File: main_menu.html [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    <head>
+        <title>Flappy Main Menu</title>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="icon" type="image/ico" href="images/favicon.ico"/>
+        <link rel="stylesheet" type="text/css" href="style/main_menu.css">
+    </head>
+    <body>
+        <body style="margin:0;padding:0;border:0;">
+            <center>
```
Comment (niccokunzmann) [inline main_menu.html]: Just a note: center is 'outdated' as all style information should go into the css file. 

    body {
      text-align: center;
    }

You do not need to change this.

File: leaderboard.html [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+		<meta content='width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0' name='viewport' />
+ 	   <meta http-equiv="Content-Type" content="text/html;charset=UTF-8">
+    	<link rel="icon" type="image/ico" href="images/favicon.ico" />
+	</head>
+	<body>
+		<p><h1>This Page displays the leader board </h1></p>
+	</body>
+
+
+>>>>>>> 309b302... added main menu
```
Comment (niccokunzmann) [inline leaderboard.html]: These lines are created through merges, when two versions have conflicts you can review the files and merge them manually. I mark them with x.

File: leaderboard.html [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="icon" type="image/ico" href="images/favicon.ico" />
+    </head>
+    <body>
+        <p>
+        <h1>This Page displays the leader board </h1>
+        </p>
+    </body>
+</html>
+=======
```
Comment (niccokunzmann) [inline leaderboard.html]: x

File: leaderboard.html [hunk 7]
```diff
@@ -0,0 +1,27 @@
+<html>
+<<<<<<< HEAD
```
Comment (niccokunzmann) [inline leaderboard.html]: x

File: howtoplay.html [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+		<meta content='width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0' name='viewport' />
+ 	   <meta http-equiv="Content-Type" content="text/html;charset=UTF-8">
+    	<link rel="icon" type="image/ico" href="images/favicon.ico" />
+	</head>
+	<body>
+		<p><h1>This Page contains the game rules </h1></p>
+	</body>
+
+
+>>>>>>> 309b302... added main menu
```
Comment (niccokunzmann) [inline howtoplay.html]: x

File: howtoplay.html [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="icon" type="image/ico" href="images/favicon.ico" />
+    </head>
+    <body>
+        <p>
+        <h1>This Page contains the game rules </h1>
+        </p>
+    </body>
+</html>
+=======
```
Comment (niccokunzmann) [inline howtoplay.html]: x

File: howtoplay.html [hunk 10]
```diff
@@ -0,0 +1,27 @@
+<html>
+<<<<<<< HEAD
```
Comment (niccokunzmann) [inline howtoplay.html]: x

**Comment by niccokunzmann** [comment]
@kamishettysreeja25 It looks quite good. But I notice that your merge went wrong.

- [ ] remove the lines caused by merge
- [ ] move the body style into the css file
- [ ] "EXIT" should be "LINKS"

The points you checked at the top I agree that they are done : )

**Comment by kamishettysreeja25** [comment]
@niccokunzmann 
 -[x]  removed the lines caused by merge
-[x]  removed the body style into the css file
-[x]  "Exit" should  be "Links"

I have idented all the html files as i have done main_menu.html hope you liked it :)  . I had conflicts during merging but they are resolved. I have changed the css file of main_menu.html now the main menu looks like this similar to the game window size 
screenshot from 2016-02-25 16 38 23

**Comment by niccokunzmann** [comment]
Very good. I merge it and create a new issue that the main menu should work regardless of the size of the window.

**Comment by niccokunzmann** [comment]
Thank you for the main menu! If you like, you can add your name to the credits.html file.

**Comment by kamishettysreeja25** [comment]
@niccokunzmann  Thanks a lot :) I would like to continue my work to improve the main menu 
1-howtoplay.html file 
I will add the instructions to play in a user friendly manner .
can you tell me what all should it contain?

**Comment by niccokunzmann** [comment]
@kamishettysreeja25 Can you create new issues for those things you think should be improved? It is a means of communication so other people know what you are up to. Maybe they like to help or have other ideas.

**Pull request merged**